src/sentry/db/models/manager.py

```diff
@@ -263,7 +263,7 @@ def contribute_to_class(self, model, name):
         super(BaseManager, self).contribute_to_class(model, name)
         class_prepared.connect(self.__class_prepared, sender=model)
 
-    def get_from_cache(self, **kwargs):
+    def get_from_cache(self, **kwargs) -> Model:  # TODO(typing): Properly type this
```

Comment:
We might be able to hook into whatever django-stubs is doing to get this to work.

Reply:
I don't think you can type this without mypy plugins so I would not bother

Reply:
Is there some reason we wouldn't want to use mypy plugins? I think django-stubs is a plugin as well

Reply:
pyright doesn't support them